orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/client/SaltConnector.java

```diff
@@ -245,6 +236,29 @@ public GenericResponses upload(Iterable<String> targets, String path, String fil
         return getGenericResponses(targets, path, fileName, content, distributeResponse);
     }
 
+    private void addExtraArguments(SaltClientType clientType, Form form, String[] arg) {
```

Comment:
could you return with the `form` so it would be visible that the method is modifying it?

Reply:
Done.